### ISSUE TITLE
Final Peer Review smh367

### ISSUE THREAD

**Issue by sawhuang** [mchao409/ORIE-4741-Class-Project#14]
The goal of the project was to predict the number of bikes that will be rented from a specific station in a certain zip code each hour in San Francisco. They used a dataset that compiled information on zip codes, weather, number of bikes rented, and other features that may have had an effect on the number of bikes rented. This project hoped to provide people with the best times to rent a bike and where in the San Francisco Bay Area.

Things I like:
1.) I like that your group was realistic and admitted that it would not be best to put this model into production as the data may be corrupted. I also like how you handled identifiably corrupted data by using the Google Maps API.
2.) I like that you attempted to use various models such as linear, lasso, and ridge regression with various regularizers to test if you overfitted.
3.) I like that you managed to achieve a very high test accuracy with your final model (decision trees).

Ways to improve:
1.) I think your analysis would have benefitted from more visualizations. The visualizations you have do a good job at describing the data, but you can improve by using different types of visualizations, different colors, and different data such as model accuracies as you changed parameters.
2.) I think your analysis would have benefitted from testing more models. Your decision tree classifier achieved very high accuracy, but you never explored other classifiers such as random forest, logistic regression, and neural networks.
3.) I think your analysis would have benefitted from a thorough examination of which features seemed to be most significant in your predictions. It would add some interesting insight into your data.
